Makefile

```diff
@@ -53,9 +44,28 @@ ifdef KEYTOOL
 			-storepass 'changeit' \
 			-file "$${pem_file}" ; \
 	done
-	$(CP) $(KEYSTORE) $(DESTDIR)
-	$(CHOWN) $(USER):$(GROUP) $(DESTDIR)/$(KEYSTORE)
-	$(CHMOD) $(PERMISSIONS) $(DESTDIR)/$(KEYSTORE)
+	$(RM) ./pl_*.pem
 else
 	$(error not creating keystore, keytool cannot be found)
 endif
+
+prepare: clean bundle keystore
+
+refresh-certs: clean
+	$(RM) *.crt
+	perl mk-ca-bundle.pl
+	python certdata2pem.py
```

Comment:
I think with the update this script got in the recent PR it now requires `python3` https://github.com/puppetlabs/puppet-ca-bundle/pull/17#pullrequestreview-1006420471

Reply:
I _think_ most of the time `python` will default to Python 3 since it was released so long ago, I know Python 2.7 was removed entirely from macOS in a recent update.

Reply:
Since this is meant to be run on dev laptops that works for me. Though if this goes into Jenkins, RHEL 9 and Ubuntu 22.04 are the first enterprise linux versions to move `python` to point to `python3` by default.